src/main/java/com/nirmata/workflow/details/internalmodels/StartedTask.java

```diff
@@ -23,11 +23,13 @@
 {
     private final String instanceName;
     private final LocalDateTime startDateUtc;
+    private int progress;
```

Comment:
Thanks for your comment Jordan. I've made those changes in my workspace during the testing, and I'm gonna submit it soon.